swatch-product-configuration/src/main/resources/subscription_configs/RHEL/RHEL_for_x86_ha.yaml

```diff
@@ -0,0 +1,15 @@
+---
+platform: RHEL
+
+id: rhel-for-x86-ha
+
+variants:
+  - tag: rhel-for-x86-ha
+    engineeringIds:
+      - 83 # Red Hat Enterprise Linux High Availability for x86_64
```

Comment:
This link was helpful here: https://offering-manager.corp.redhat.com/offerings/view/MCT0907?child=SVC0099#eng-products